control-center/Dockerfile.ubi8

```diff
@@ -52,7 +52,7 @@ RUN echo "===> Installing ${COMPONENT}..." \
     && rpm --import ${CONFLUENT_PACKAGES_REPO}/archive.key \
     && printf "[Confluent.dist] \n\
 name=Confluent repository (dist) \n\
-baseurl=${CONFLUENT_PACKAGES_REPO}/7 \n\
+baseurl=${CONFLUENT_PACKAGES_REPO} \n\
 gpgcheck=1 \n\
 gpgkey=${CONFLUENT_PACKAGES_REPO}/archive.key \n\
 enabled=1 \n\
```

Comment:
Just remove this whole repo definition - Doing the change this way makes it redundant to the one bellow.

Reply:
I actually have the same question since I followed https://docs.confluent.io/platform/current/installation/installing_cp/rhel-centos.html.
```
You must have the entries for both repositories, [Confluent.dist] and [Confluent], as shown below.
```
Or I guess it does not matter?

Reply:
Keep in mind that this is master, an un-released "version" - this page will change for 7.0.0 and 7.1.0 from https://github.com/confluentinc/docs-platform/pull/983/files

Reply:
One could also argue that customers deriving from our images rely on this aspect, to which I would say then to change things in the way that the PR https://github.com/confluentinc/docs-platform/pull/983 says, which is to change the repo baseurl endpoint to the new one.